### PR TITLE
fix(editor): Pin all data regardless of pagination

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/components/RunData.vue [hunk 1]
```diff
@@ -434,7 +434,8 @@
 				!hasRunError &&
 				binaryData.length === 0 &&
 				dataCount > pageSize &&
-				!isSchemaView
+				!isSchemaView &&
+				!pinData
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/RunData.vue:438]: As far as I know, `pinData` still needs to be paginated.

File: packages/editor-ui/src/components/RunData.vue [hunk 2]
```diff
@@ -809,8 +810,8 @@ export default defineComponent({
 					  ];
 			}
 
-			// We don't want to paginate the schema view
-			if (this.isSchemaView) {
+			// We don't want to paginate the schema view and pinned data
+			if (this.isSchemaView || this.pinData) {
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/RunData.vue:814]: Same as above.

**Review by alexgrozav — CHANGES_REQUESTED** [review]
As far as I know, `pinData` still needs to be paginated. Other than that, the fix works as expected.

**Review by alexgrozav — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.232.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.232.0)

**Pull request merged**